exporter/prometheusremotewriteexporter/metadata.yaml

```diff
@@ -27,3 +27,17 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    exporter_prometheusremotewrite_consumers:
+      enabled: true
+      description: Number of configured workers to use to fan out the outgoing requests
+      unit: "1"
```

Comment:
```suggestion
      unit: "{consumer}"
```

Or {worker} could work as well

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37912/commits/62bcf74b1d50eb487fe26481f73b173b42e1d15d